.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<h4> Summary (Short summary of what is being done) : </h4>
+<br>
+<h4> Description (Describe in detail the fix made) : </h4>
+<br>
+<h4> Bug ID (if any) : </h4>
+<br>
+<h4> Public Documentation (if any) : </h4>
+<br>
+<h4> TESTED (Test Cases with scenario and description - must have 1 positive and 1 negative scenario) : </h4>
```

Comment:
Can we include a checkbox to ensure all unit test cases are passing? Taking reference from 'java-code-samples' repository.

Reply:
Won't CI fail if any unit-test is failing ultimately blocking PR submission?

Reply:
Also we don't have any unit test cases now unlike java-code-samples